app/main.py

```diff
@@ -1,8 +1,39 @@
 class Person:
-    # write your code here
-    pass
+    # Class attribute to store Person instances by name
+    people: dict[str, "Person"] = {}
```

Comment:
Using the `Person.people` dictionary to store instances by name can lead to unexpected behavior if multiple instances with the same name are created. Consider using a different approach to manage instances if unique names cannot be guaranteed.